src/node/app/static.ts

```diff
@@ -32,17 +31,25 @@ export class StaticHttpProvider extends HttpProvider {
   /**
    * Return a resource with variables replaced where necessary.
    */
-  protected async getReplacedResource(route: Route): Promise<HttpResponse> {
+  protected async getReplacedResource(request: http.IncomingMessage, route: Route): Promise<HttpResponse> {
     // The first part is always the commit (for caching purposes).
```

Comment:
Shouldn't this be removed.

Reply:
The comment? I think it's still accurate, the first part is sliced off because it's the commit.

Reply:
Oh just saw your comment below.